build/_build/Build.Steps.cs

```diff
@@ -37,6 +37,7 @@ partial class Build
     AbsolutePath ArtifactsDirectory => Artifacts ?? (OutputDirectory / "artifacts");
     AbsolutePath WindowsTracerHomeZip => ArtifactsDirectory / "windows-tracer-home.zip";
     AbsolutePath BuildDataDirectory => RootDirectory / "build_data";
+    AbsolutePath LibSqreenDirectory => RootDirectory / "packages" / "libsqreen.1.1.2.2";
```

Comment:
I assume we have a package reference somewhere that pulls down the nuget package?

Reply:
It's a good point, it was only referenced in the samples, as we only need it at runtime, on to build against, I've added it to packages.config in Datadog.Trace.ClrProfiler.Native. Another way of accessing may be need for the Linux installer, but that's the next task.